src/pages/projects/projects.tsx

```diff
@@ -0,0 +1,7 @@
+import { ProjectsList } from "@/modules/projects";
+
+const ProjectsPage = () => {
+    return <ProjectsList />;
```

Comment:
Breadcrumb component/service already exists, so can add smth like  `useGenericBreadcrumbs(t("Projects"))`  ( where `t` is from translation, and `useGenericBreadcrumbs` is an already existing hook) here